src/dbConfig.ts

```diff
@@ -0,0 +1,7 @@
+export const dbConfig = {
```

Comment:
In the current structure, you have the folder for database stuff so it is better to put it there